tests/integration/commands/deploy/deploy.test.js

```diff
@@ -509,7 +509,14 @@ describe.skipIf(process.env.NETLIFY_TEST_DISABLE_LIVE === 'true').concurrent('co
         .withContentFile({
           content: `
           export default async () => new Response("Internal V2 API")
-          export const config = { path: "/internal-v2-func" }
+          export const config = {
+            path: "/internal-v2-func",
+            rateLimit: {
```

Comment:
I don't see any assertions added. I'm assuming you used this for testing things locally? Fine by me, i'm wondering if there's a good way of asserting that the ratelimit config was actually applied 🤔

Reply:
yes I tested it out locally